sdlib/d/gc/tcache.d

```diff
@@ -171,6 +231,29 @@ public:
 	}
 
 private:
+	// Force large allocation rather than slab
+	size_t upsizeToLarge(size_t size) {
```

Comment:
Why is this a member method? This doesn't use any of the data that's in this struct, nor does it call any of its methods.